src/components/_basic.less

```diff
@@ -27,6 +27,7 @@
   &:active,
   &:focus {
     outline: none;
+    box-shadow: 0px 5px 0px @lightGray;
```

Comment:
What is this for?

Reply:
Actually I thought it would be better..if those buttons get back to their style...as soon as we hover-out the pointer....after clicking....bt its not working in the same way......we have to click some where else in the screen to make those button style back to their own after clicking

Reply:
Please remove any changes to files that are not related to the click button specifically. Changes tp `src/components/_basics.less` can affect all buttons